unit-tests/post-processing/test-post-processing-filters.py

```diff
@@ -0,0 +1,469 @@
+            self.temp_pb = filters_cfg.temporal_filter
+            self.temp_filter.set_option(rs.option.filter_smooth_alpha, filters_cfg.temporal_alpha)
+            self.temp_filter.set_option(rs.option.filter_smooth_delta, filters_cfg.temporal_delta)
+            self.temp_filter.set_option(rs.option.holes_fill, filters_cfg.temporal_persistence)
+
+        if filters_cfg.holes_filter:
+            self.holes_pb = filters_cfg.holes_filter
+            self.hole_filling_filter.set_option(rs.option.holes_fill, filters_cfg.holes_filling_mode)
+
+    def process(self, frame_input):
+        processed = frame_input
+        if self.dec_pb:
+            processed = self.dec_filter.process(processed)
+
+        processed = self.depth_to_disparity.process(processed)
+
+        if self.spat_pb:
+            processed = self.spat_filter.process(processed)
+
+        if self.temp_pb:
+            processed = self.temp_filter.process(processed)
+
+        processed = self.disparity_to_depth.process(processed)
+
+        if self.holes_pb:
+            processed = self.hole_filling_filter.process(processed)
+
+        return processed
+
+class ppf_test_config:
+    def __init__(self):
+        self.name = ""
+        self.spatial_filter = False
+        self.spatial_alpha = 0.0
+        self.spatial_delta = 0
+        self.spatial_iterations = None
+        self.temporal_filter = False
+        self.temporal_alpha = 0.0
+        self.temporal_delta = 0
+        self.temporal_persistence = 0
+        self.holes_filter = False
+        self.holes_filling_mode = 0
+        self.downsample_scale = 1
+        self.depth_units = 0.001
+        self.stereo_baseline_mm = 0.0
+        self.focal_length = 0.0
+        self.input_res_x = 0
+        self.input_res_y = 0
+        self.output_res_x = 0
+        self.output_res_y = 0
+        self.frames_sequence_size = 1
+        self.input_frame_names = []
+        self.output_frame_names =[]
+        self._input_frames = []
+        self._output_frames = []
+
+    def reset(self):
+        self.name = ""
+        self.spatial_filter = self.temporal_filter = self.holes_filter = False
+        self.spatial_alpha = self.temporal_alpha = 0
+        self.spatial_iterations = self.temporal_persistence = self.holes_filling_mode = self.spatial_delta = self.temporal_delta = 0
+        self.input_res_x = self.input_res_y = self.output_res_x = self.output_res_y = 0
+        self.downsample_scale = 1
+        self._input_frames = []
+        self._output_frames = []
+
+
+def load_from_binary(file_name):
+    with open(file_name, 'rb') as binary_file:
+        loaded_data = binary_file.read()
+    return loaded_data
+
+
+def attrib_from_csv(file_path):
+    global metadata_attributes
+
+    with open(file_path, 'r') as csv_file:
+        reader = csv.reader(csv_file)
+        dict = {row[0]: row[1] for row in reader}
+
+    cfg = ppf_test_config()
+
+    cfg.input_res_x = int(dict.get(metadata_attributes.get('res_x'), 0))
```

Comment:
What is the benefit of using `metadata_attributes` map over the value strings directly?

Reply:
Originally in the cpp tests the res_x, res_y etc were enums and then they used the map to get the string values. I can remove the map and use the actual longer strings. Tell me what you think.

Reply:
I think it's redundant and can be removed.

Reply:
Removed